packages/@aws-cdk/aws-gamelift/lib/matchmaking-ruleset.ts

```diff
@@ -105,7 +105,7 @@ export abstract class MatchmakingRuleSetBase extends cdk.Resource implements IMa
       namespace: 'AWS/GameLift',
       metricName: metricName,
       dimensionsMap: {
-        FleetId: this.matchmakingRuleSetName,
+        MatchmakingRuleSetName: this.matchmakingRuleSetName,
```

Comment:
not sure I quite understand this change. Is this just changing the name of the identifier to be more pedantic/specific?

Reply:
:+1: to @HBobertz's comment, I also don't understand this.

Reply:
It was a typo, DimensionsMap was incorrect before so I fixed it by using a MatchmakingRuleSetName instead of a FleetId